src/controller/java/AndroidOperationalCredentialsIssuer.cpp

```diff
@@ -219,17 +221,19 @@ CHIP_ERROR AndroidOperationalCredentialsIssuer::CallbackGenerateNOCChain(const B
     JniReferences::GetInstance().N2J_ByteArray(env, attestationChallenge.data(), attestationChallenge.size(),
                                                javaAttestationChallenge);
 
-    const ByteSpan & attestationElements = mAutoCommissioner->GetCommissioningParameters().GetAttestationElements().Value();
+    const ByteSpan & attestationElements = mAutoCommissioner->GetAttestationElements();
     jbyteArray javaAttestationElements;
     JniReferences::GetInstance().N2J_ByteArray(env, attestationElements.data(), attestationElements.size(),
                                                javaAttestationElements);
+    // new log message below
+    ChipLogError(Controller, "AndroidOpCredsIssuer attestationElements size %d/%d", static_cast<int>(attestationElements.size()),
```

Comment:
Please remove this internal debug

Reply:
changed to ChipLogProgress

Reply:
This line should completely be removed, and the comment above as well

Reply:
removed